projects/kit/components/input-date/input-date.component.ts

```diff
@@ -148,10 +149,28 @@ export class TuiInputDateComponent
     }
 
     get computedMin(): TuiDay {
+        /**
+         * TODO: we can delete this workaround in v4.0
+         * after solving this issue:
+         * https://github.com/taiga-family/maskito/issues/604
+         */
+        if (this.value && this.control?.pristine) {
+            return this.options.min;
```

Comment:
Maybe it's better to direcrly return `TUI_FIRST_DAY`/`TUI_LAST_DAY`, because if provide option through DI, it will trigger value changes - and we don't want that

@n-barsukov what do you think about it?

Reply:
Yep, I think `TUI_FIRST_DAY` / `TUI_LAST_DAY` is better 👍

Reply:
Updated